apollo-ios/Sources/ApolloAPI/Selection.swift

```diff
@@ -2,9 +2,11 @@ public enum Selection {
   /// A single field selection.
   case field(Field)
   /// A fragment spread of a named fragment definition.
-  case fragment(any Fragment.Type, deferred: Condition? = nil)
+  case fragment(any Fragment.Type)
   /// An inline fragment with a child selection set nested in a parent selection set.
-  case inlineFragment(any InlineFragment.Type, deferred: Condition? = nil)
+  case inlineFragment(any InlineFragment.Type)
+
+  case deferred(if: Condition? = nil, any Deferrable.Type, label: String)
```

Comment:
Can we add inline documentation here?

Reply:
Should `label` be optional since we don't use it for named fragments always now?

Reply:
> Should label be optional since we don't use it for named fragments always now?

Yes with my merged PR from yesterday this will need to become optional - good catch.